dnf/cli/cli.py

```diff
@@ -807,6 +807,9 @@ def configure(self, args, option_parser=None):
         if opts.sleeptime is not None:
             time.sleep(random.randrange(opts.sleeptime * 60))
 
+        if opts.downloaddir is not None:
+            self.base.conf.downloaddir = opts.downloaddir
```

Comment:
I think that it is not good idea to create new conf option or property here. If there will be no that option, the call of self.conf.downloaddir will result in error.